modules/ossm-excluding-namespaces-from-cluster-wide-mesh-console.adoc

```diff
@@ -4,9 +4,14 @@
 :_mod-docs-content-type: PROCEDURE
 [id="ossm-excluding-namespaces-from-cluster-wide-mesh-console_{context}"]
 = Including and excluding namespaces from a cluster-wide mesh by using the web console
- 
-By default, the {SMProductName} Operator uses discovery selectors to identify the namespaces that make up the mesh. Namespaces that do not contain the label defined in the `ServiceMeshMemberRoll` resource are not matched by the discovery selector and are excluded from the mesh.  
- 
+
+In a cluster-wide mesh, you can add discovery selectors to the `ServiceMeshControlPlane` resource. Discovery selectors define which namespaces the control plane can discover. The control plane ignores any namespace that does not match one of the discovery selectors, which excludes the namespace from the mesh.
```

Comment:
Thought: you probably want to indicate that you're using the web UI here, just as you'd want to indicate that you're using the CLI in the description of the other module.

Reply:
@maxwelldb, the title for each procedures indicates if you are using the console or the CLI. I thought this was sufficient. Can you give me an example of what you had in mind in terms of clarifying this?

Reply:
I don't typically work on things that have GUIs, but I see this sort of language pretty frequently in the docs for tasks that can be accomplished by using different UIs:

- https://docs.openshift.com/container-platform/4.15/nodes/pods/nodes-pods-autoscaling.html#nodes-pods-autoscaling-creating-web-console_nodes-pods-autoscaling

- https://docs.openshift.com/container-platform/4.15/nodes/pods/nodes-pods-autoscaling.html#nodes-pods-autoscaling-creating-cpu_nodes-pods-autoscaling

Not that pattern verbatim, of course. Just calling out in the description how you're accomplishing the task.

Reply:
Thank you, I've made the change. I appreciate your suggestions.